03 React Basics and working with component/01-starting-setup/src/App.js

```diff
@@ -1,7 +1,54 @@
+// Coding Exercise 3 : Working with JSX Code
+
+// Task
+
+// 1. This is a quick exercise to get started with React and JSX code: 
+//    Your task is to edit the JSX code of a provided React component such that
+//    it displays a h1 element with the text "Exercise done!" on the page.
+
+// Important: The custom JSX code must be added inside the existing <div> element.
+
+import React from "react";
+import ExerciseComponent from './components/ExerciseComponent';
+import Product from './components/products/Product';
+
 function App() {
+  const products = [
```

Comment:
can you create utils.js file and declare all the constants there and use them here. instead of declaring it in the App.js